File: viajerosrepoangular-master/viajerosrepoangular-master/viajeros/src/app/models/Chat/IsChoferDto.ts
export class IsChoferDto {
    ischofer: boolean;       // Indicates if the user is the driver
    idChofer: number;        // ID of the driver
    nombreChofer: string;    // Name of the driver
    idPasajero: number | null; // ID of the passenger (null if the user is the driver)
    nombrePasajero: string | null; // Name of the passenger (null if the user is the driver)
  
    constructor(
      ischofer: boolean, 
      idChofer: number, 
      nombreChofer: string, 
      idPasajero: number | null, 
      nombrePasajero: string | null
    ) {
      this.ischofer = ischofer;
      this.idChofer = idChofer;
      this.nombreChofer = nombreChofer;
      this.idPasajero = idPasajero;
      this.nombrePasajero = nombrePasajero;
    }
  }
  

File: viajerosrepojava-master/viajerosrepojava-master/viajeros/viajeros/src/main/java/ps/com/viajeros/dtos/chat/IsChoferDto.java
package ps.com.viajeros.dtos.chat;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class IsChoferDto {
    private boolean ischofer; // Contenido del mensaje
    private Long idChofer;
    private String nombreChofer;// ID del mensaje generado en la base de datos
    private Long idPasajero;   // ID del usuario que envía el mensaje
    private String nombrePasajero;
}
